.vsts-ci/releaseBuild.yml

```diff
@@ -1,3 +1,23 @@
+parameters:
+- name: release
+  type: string
+  displayName: |
+    Use `stable` to release by default.
+
+    Use `private` if you want to create a branch on the store to test the package.
+    This will create a branch under the edge branch automatically that are difficult, but not impossible to find.
+
+    `candidate`, `beta`, and `edge` are public but we don't have any existing meaning for these channels.
+  values:
+      - private
+      - stable
+      - candidate
```

Comment:
if we don't have any meaning for candidate, beta and edge does it make sense to remove these til we do have meaning for those channels? like what happens if someone tries to use/specify those values?

Reply:
They are possible values for public releases... I used edge to test the pipeline.  Let's leave them as they have definite meanings in snap.